File: trunk/quote/Projects/SampleProperties/OtherProperiesFactory.cs
using System;

using Model.Template;
using Host;

namespace SampleProperties
{
    [Register(Key = typeof(Model.Template.Ext.IOtherProperiesFactory))]
    public class OtherProperiesFactory : Model.Template.Ext.IOtherProperiesFactory
    {
        public OtherProperiesFactory()
        {
        }

        public Model.Common.SaveableProperties CreateOtherProperties(
            Header header, int id)
        {
            return new SampleOtherProperties(header);
        }

    }
}

File: trunk/quote/Projects/DekalbProperties/OtherProperiesFactory.vb
Imports PluginHost

<Register(Key:=GetType(Model.Template.Ext.IOtherProperiesFactory))>
Public Class OtherProperiesFactory
    Implements Model.Template.Ext.IOtherProperiesFactory

    Public Function CreateOtherProperties(header As Model.Template.Header, _
                                          id As Integer) As  _
                                      Model.Common.SaveableProperties _
                                      Implements Model.Template.Ext.IOtherProperiesFactory.CreateOtherProperties
        Return New DekalbOtherProperties(header)
    End Function

End Class
